src/handleErrors.js

```diff
@@ -0,0 +1,34 @@
+const handleErrors = (text, caseName) => {
+  const caseNames = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message:
+        'Text to convert is required. Correct request is: '
```

Comment:
It's a bad way to represent string. If you have trouble with linter, you can add eslint-disable rule or change eslint rule.